docs/running.rst

```diff
@@ -65,6 +65,39 @@ You may also want to enable human readable logs in the shared config file:
     ; Uncomment to provide stdout logging in a more human readable format.
     #human_logs
 
+Configuring SSL/TLS
+-------------------
+
+The Push protocol mandates use of SSL/TLS connections to ensure the integrity
+of push subscription data. This rule can be ignored for testing purposes
+only. This requires a valid, trusted TLS certificate (for example, from `Let's
+Encrypt`_).
+
+The ``ssl_key`` and ``ssl_cert`` options configure a TLS certificate for both
+the endpoint's HTTPS and connection node's WebSocket Secure (WSS) connections:
+
```

Comment:
Wonder if we should note that different creds can be used for endpoint and connection handlers by moving these option definitions to the appropriate config files? (Could be consideration if you don't have a wild card cert.)